channels/do.m3u

```diff
@@ -3,6 +3,8 @@
 https://ss6.domint.net:3028/ame_str/amecanal47/playlist.m3u8
 #EXTINF:-1 tvg-id="AnimeZoneTV.do" tvg-country="LATAM;PT;ES" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/AnimeZoneXTV/picture?width=300&height=300" group-title="Animation",Anime Zone TV (480p) [Not 24/7]
 http://azxtv.com/hls/stream.m3u8
+#EXTINF:-1 tvg-id="AkíTV.do" tvg-country="DO" tvg-language="" tvg-logo="" group-title="",AkíTV
+https://cm.hostlagarto.com:4445/Akitv/Akitv.myStream/playlist.m3u8
```

Comment:
Can't view it from here. Maybe others can.
https://cm.hostlagarto.com:4445/Akitv/Akitv.myStream/playlist.m3u8: Server returned 404 Not Found

Reply:
Might be not 24/7.

Reply:
I see this channel hours ago. So this channel is tagged Not 24/7.